### PR TITLE
Feature/story format 3

### PR DESCRIPTION
**Pull request by r3dDoX**
Also fixed a few attribute defaults that got overwritten from the lit-analyzer change

### CODE REVIEW

File: web-components/src/components/table/table.stories.ts [hunk 1]
```diff
@@ -71,469 +71,468 @@ const Template: StoryFn<Table> = ({
...
-    },
-  ] as ColumnDef<Person>[],
+export const RowGrouping: StoryObj<Table> = {
+  render: Template,
+  args: {
+    sortable: true,
+    filterable: true,
+    useRowGrouping: true,
+    data: expandablePersonData.slice(0, 3),
+  },
```
Comment (culas) [inline web-components/src/components/table/table.stories.ts:322]: **issue:** `columns` got lost here

File: web-components/src/components/button/button.stories.ts [hunk 2]
```diff
@@ -54,73 +54,95 @@ const Template: StoryFn<Button> = ({
...
 `;
 
-export const Default = Template.bind({});
-Default.args = {
-  slot: 'Primary Button',
+export const Default: StoryObj<Button> = {
+  render: Template,
+  args: {
+    slot: 'Primary Button',
+  },
```
Comment (culas) [inline web-components/src/components/button/button.stories.ts:61]: **praise:** these are so much nicer and cleaner

Is this due to a Storybook update or could have had it this way all along?

**Review by culas — CHANGES_REQUESTED** [review]

File: web-components/src/components/button/button.stories.ts (hunk 2 above)
Comment (r3dDoX) [reply web-components/src/components/button/button.stories.ts:61]: This new format was released at some point towards the end of last year

Comment (r3dDoX) [reply web-components/src/components/button/button.stories.ts:61]: I also think it's nice and clean. @Cybertron01Z mentioned it first :)

File: web-components/src/components/table/table.stories.ts (hunk 1 above)
Comment (r3dDoX) [reply web-components/src/components/table/table.stories.ts:347]: Good catch! Readded them :+1:

**Review by culas — APPROVED** [review]

**Pull request merged**